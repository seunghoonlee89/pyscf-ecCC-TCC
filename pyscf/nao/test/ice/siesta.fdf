#************************************************************************************
#*                   Generated by cif2cell 1.1.5 2014-04-17 6:36                    *
#*  T. Bjorkman, Comp. Phys. Commun. 182, 1183-1186 (2011). Please cite generously. *
#*                                                                                  *
#*                Data obtained from COD. Reference number : 1011023                *
#*                            H2 O   (Dihydrogen oxide)                             *
#*    Bernal, J D and Fowler, R H, Journal of Chemical Physics 1, 515-548 (1933)    *
#************************************************************************************
AtomicCoordinatesFormat     Fractional
LatticeConstant             7.82 Ang
NumberOfAtoms               36
NumberOfSpecies             2
SpinPolarized               .true.
%block LatticeVectors
  0.866025403784439  -0.500000000000000   0.000000000000000 
  0.000000000000000   1.000000000000000   0.000000000000000 
  0.000000000000000   0.000000000000000   0.941176470588235 
%endblock LatticeVectors
%block AtomicCoordinatesAndAtomicSpecies
  0.333333333333333   0.000000000000000   0.174000000000000    1
  0.666666666666667   0.000000000000000   0.674000000000000    1
  0.333333333333333   0.333333333333333   0.674000000000000    1
  0.000000000000000   0.333333333333333   0.174000000000000    1
  0.000000000000000   0.666666666666667   0.674000000000000    1
  0.666666666666667   0.666666666666667   0.174000000000000    1
  0.438000000000000   0.000000000000000   0.026000000000000    1
  0.562000000000000   0.000000000000000   0.526000000000000    1
  0.438000000000000   0.438000000000000   0.526000000000000    1
  0.000000000000000   0.438000000000000   0.026000000000000    1
  0.000000000000000   0.562000000000000   0.526000000000000    1
  0.562000000000000   0.562000000000000   0.026000000000000    1
  0.772000000000000   0.105000000000000   0.975000000000000    1
  0.228000000000000   0.895000000000000   0.475000000000000    1
  0.667000000000000   0.895000000000000   0.975000000000000    1
  0.667000000000000   0.772000000000000   0.475000000000000    1
  0.105000000000000   0.772000000000000   0.975000000000000    1
  0.333000000000000   0.105000000000000   0.475000000000000    1
  0.105000000000000   0.333000000000000   0.475000000000000    1
  0.333000000000000   0.228000000000000   0.975000000000000    1
  0.895000000000000   0.667000000000000   0.975000000000000    1
  0.895000000000000   0.228000000000000   0.475000000000000    1
  0.228000000000000   0.333000000000000   0.975000000000000    1
  0.772000000000000   0.667000000000000   0.475000000000000    1
  0.333333333333333   0.000000000000000   0.062500000000000    2
  0.666666666666667   0.000000000000000   0.562500000000000    2
  0.333333333333333   0.333333333333333   0.562500000000000    2
  0.000000000000000   0.333333333333333   0.062500000000000    2
  0.000000000000000   0.666666666666667   0.562500000000000    2
  0.666666666666667   0.666666666666667   0.062500000000000    2
  0.666666666666667   0.000000000000000   0.937500000000000    2
  0.333333333333333   0.000000000000000   0.437500000000000    2
  0.666666666666667   0.666666666666667   0.437500000000000    2
  0.000000000000000   0.666666666666667   0.937500000000000    2
  0.000000000000000   0.333333333333333   0.437500000000000    2
  0.333333333333333   0.333333333333333   0.937500000000000    2
%endblock AtomicCoordinatesAndAtomicSpecies
%block ChemicalSpeciesLabel
1       1       H       
2       8       O       
%endblock ChemicalSpeciesLabel

%block kgrid_Monkhorst_Pack
2 0 0 0.0
0 2 0 0.0
0 0 2 0.0
%endblock kgrid_Monkhorst_Pack

PAO.EnergyShift  200 meV

DM.UseSaveDM   .true.
DM.NumberPulay  3
DM.Tolerance    1e-4
DM.MixingWeight 0.5

### Molecular dynamics 
ElectronicTemperature      300 K
MD.TypeOfRun               CG
MD.NumCGsteps              0
MaxSCFIterations           1000

DM.UseSaveDM .true.

# Write .WFSX and .HSX files
COOP.Write     .true.
# Write .DIM and .PLD files
WriteDenchar   .true.
