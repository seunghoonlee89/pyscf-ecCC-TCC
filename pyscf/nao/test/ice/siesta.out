Siesta Version: siesta-4.0b-494
Architecture  : x86_64-mkl_mpi
Compiler flags: mpif90 -I.  -ffixed-line-length-0    -g -O2
PP flags      : -DFC_HAVE_FLUSH -DFC_HAVE_ABORT -DMPI 
PARALLEL version

* Running on    4 nodes in parallel
>> Start of run:  10-JUL-2017  19:36:54

                           ***********************       
                           *  WELCOME TO SIESTA  *       
                           ***********************       

reinit: Reading from standard input
************************** Dump of input data file ****************************
#************************************************************************************
#*                   Generated by cif2cell 1.1.5 2014-04-17 6:36                    *
#*  T. Bjorkman, Comp. Phys. Commun. 182, 1183-1186 (2011). Please cite generously. *
#*                                                                                  *
#*                Data obtained from COD. Reference number : 1011023                *
#*                            H2 O   (Dihydrogen oxide)                             *
#*    Bernal, J D and Fowler, R H, Journal of Chemical Physics 1, 515-548 (1933)    *
#************************************************************************************
AtomicCoordinatesFormat     Fractional
LatticeConstant             7.82 Ang
NumberOfAtoms               36
NumberOfSpecies             2
SpinPolarized               .true.
%block LatticeVectors
  0.866025403784439  -0.500000000000000   0.000000000000000
  0.000000000000000   1.000000000000000   0.000000000000000
  0.000000000000000   0.000000000000000   0.941176470588235
%endblock LatticeVectors
%block AtomicCoordinatesAndAtomicSpecies
  0.333333333333333   0.000000000000000   0.174000000000000    1
  0.666666666666667   0.000000000000000   0.674000000000000    1
  0.333333333333333   0.333333333333333   0.674000000000000    1
  0.000000000000000   0.333333333333333   0.174000000000000    1
  0.000000000000000   0.666666666666667   0.674000000000000    1
  0.666666666666667   0.666666666666667   0.174000000000000    1
  0.438000000000000   0.000000000000000   0.026000000000000    1
  0.562000000000000   0.000000000000000   0.526000000000000    1
  0.438000000000000   0.438000000000000   0.526000000000000    1
  0.000000000000000   0.438000000000000   0.026000000000000    1
  0.000000000000000   0.562000000000000   0.526000000000000    1
  0.562000000000000   0.562000000000000   0.026000000000000    1
  0.772000000000000   0.105000000000000   0.975000000000000    1
  0.228000000000000   0.895000000000000   0.475000000000000    1
  0.667000000000000   0.895000000000000   0.975000000000000    1
  0.667000000000000   0.772000000000000   0.475000000000000    1
  0.105000000000000   0.772000000000000   0.975000000000000    1
  0.333000000000000   0.105000000000000   0.475000000000000    1
  0.105000000000000   0.333000000000000   0.475000000000000    1
  0.333000000000000   0.228000000000000   0.975000000000000    1
  0.895000000000000   0.667000000000000   0.975000000000000    1
  0.895000000000000   0.228000000000000   0.475000000000000    1
  0.228000000000000   0.333000000000000   0.975000000000000    1
  0.772000000000000   0.667000000000000   0.475000000000000    1
  0.333333333333333   0.000000000000000   0.062500000000000    2
  0.666666666666667   0.000000000000000   0.562500000000000    2
  0.333333333333333   0.333333333333333   0.562500000000000    2
  0.000000000000000   0.333333333333333   0.062500000000000    2
  0.000000000000000   0.666666666666667   0.562500000000000    2
  0.666666666666667   0.666666666666667   0.062500000000000    2
  0.666666666666667   0.000000000000000   0.937500000000000    2
  0.333333333333333   0.000000000000000   0.437500000000000    2
  0.666666666666667   0.666666666666667   0.437500000000000    2
  0.000000000000000   0.666666666666667   0.937500000000000    2
  0.000000000000000   0.333333333333333   0.437500000000000    2
  0.333333333333333   0.333333333333333   0.937500000000000    2
%endblock AtomicCoordinatesAndAtomicSpecies
%block ChemicalSpeciesLabel
1       1       H
2       8       O
%endblock ChemicalSpeciesLabel
%block kgrid_Monkhorst_Pack
2 0 0 0.0
0 2 0 0.0
0 0 2 0.0
%endblock kgrid_Monkhorst_Pack
PAO.EnergyShift  200 meV
DM.UseSaveDM   .true.
DM.NumberPulay  3
DM.Tolerance    1e-4
DM.MixingWeight 0.5
### Molecular dynamics
ElectronicTemperature      300 K
MD.TypeOfRun               CG
MD.NumCGsteps              0
MaxSCFIterations           1000
DM.UseSaveDM .true.
# Write .WFSX and .HSX files
COOP.Write     .true.
# Write .DIM and .PLD files
WriteDenchar   .true.
************************** End of input data file *****************************

reinit: -----------------------------------------------------------------------
reinit: System Name: 
reinit: -----------------------------------------------------------------------
reinit: System Label: siesta                                                      
reinit: -----------------------------------------------------------------------

initatom: Reading input for the pseudopotentials and atomic orbitals ----------
 Species number:            1  Label: H Atomic number:           1
 Species number:            2  Label: O Atomic number:           8
Ground state valence configuration:   1s01
Reading pseudopotential information in formatted form from H.psf

Valence configuration for pseudopotential generation:
1s( 1.00) rc: 1.25
2p( 0.00) rc: 1.25
3d( 0.00) rc: 1.25
4f( 0.00) rc: 1.25
Ground state valence configuration:   2s02  2p04
Reading pseudopotential information in formatted form from O.psf

Valence configuration for pseudopotential generation:
2s( 2.00) rc: 1.14
2p( 4.00) rc: 1.14
3d( 0.00) rc: 1.14
4f( 0.00) rc: 1.14
For H, standard SIESTA heuristics set lmxkb to 2
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.
For O, standard SIESTA heuristics set lmxkb to 3
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.

<basis_specs>
===============================================================================
H                    Z=   1    Mass=  1.0100        Charge= 0.17977+309
Lmxo=0 Lmxkb= 2    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=1
          n=1  nzeta=2  polorb=1
            splnorm:   0.15000    
               vcte:    0.0000    
               rinn:    0.0000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000      0.0000    
            lambdas:    1.0000      1.0000    
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

atom: Called for H                     (Z =   1)

read_vps: Pseudopotential generation method:
read_vps: ATM3      Troullier-Martins                       
Total valence charge:    1.00000

xc_check: Exchange-correlation functional:
xc_check: Ceperley-Alder
V l=0 = -2*Zval/r beyond r=  1.2343
V l=1 = -2*Zval/r beyond r=  1.2189
V l=2 = -2*Zval/r beyond r=  1.2189
All V_l potentials equal beyond r=  1.2343
This should be close to max(r_c) in ps generation
All pots = -2*Zval/r beyond r=  1.2343

VLOCAL1: 99.0% of the norm of Vloc inside     28.493 Ry
VLOCAL1: 99.9% of the norm of Vloc inside     64.935 Ry
atom: Maximum radius for 4*pi*r*r*local-pseudopot. charge    1.45251
atom: Maximum radius for r*vlocal+2*Zval:    1.21892
GHOST: No ghost state for L =  0
GHOST: No ghost state for L =  1
GHOST: No ghost state for L =  2

KBgen: Kleinman-Bylander projectors: 
   l= 0   rc=  1.364359   el= -0.467325   Ekb= -2.005361   kbcos= -0.336422
   l= 1   rc=  1.434438   el=  0.001430   Ekb= -0.501708   kbcos= -0.021697
   l= 2   rc=  1.470814   el=  0.002365   Ekb= -0.190555   kbcos= -0.002281

KBgen: Total number of  Kleinman-Bylander projectors:    9
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:
atom: Selected multiple-zeta basis: split     

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 1s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.014700 Ry

   izeta = 1
                 lambda =    1.000000
                     rc =    5.075940
                 energy =   -0.454238
                kinetic =    0.905976
    potential(screened) =   -1.360214
       potential(ionic) =   -1.894198

   izeta = 2
                 rmatch =    3.952598
              splitnorm =    0.150000
                 energy =   -0.349543
                kinetic =    1.461076
    potential(screened) =   -1.810619
       potential(ionic) =   -2.380875

POLgen: Perturbative polarization orbital with L=  1

POLgen: Polarization orbital for state 1s

   izeta = 1
                     rc =    5.075940
                 energy =    0.658058
                kinetic =    1.324347
    potential(screened) =   -0.666290
       potential(ionic) =   -1.141575
atom: Total number of Sankey-type orbitals:  5

atm_pop: Valence configuration (for local Pseudopot. screening):
 1s( 1.00)                                                            
Vna: chval, zval:    1.00000   1.00000

Vna:  Cut-off radius for the neutral-atom potential:   5.075940

atom: _________________________________________________________________________

<basis_specs>
===============================================================================
O                    Z=   8    Mass=  16.000        Charge= 0.17977+309
Lmxo=1 Lmxkb= 3    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=2
          n=1  nzeta=2  polorb=0
            splnorm:   0.15000    
               vcte:    0.0000    
               rinn:    0.0000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000      0.0000    
            lambdas:    1.0000      1.0000    
L=1  Nsemic=0  Cnfigmx=2
          n=1  nzeta=2  polorb=1
            splnorm:   0.15000    
               vcte:    0.0000    
               rinn:    0.0000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000      0.0000    
            lambdas:    1.0000      1.0000    
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
L=3  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

atom: Called for O                     (Z =   8)

read_vps: Pseudopotential generation method:
read_vps: ATM3      Troullier-Martins                       
Total valence charge:    6.00000

xc_check: Exchange-correlation functional:
xc_check: Ceperley-Alder
V l=0 = -2*Zval/r beyond r=  1.1278
V l=1 = -2*Zval/r beyond r=  1.1278
V l=2 = -2*Zval/r beyond r=  1.1278
V l=3 = -2*Zval/r beyond r=  1.1138
All V_l potentials equal beyond r=  1.1278
This should be close to max(r_c) in ps generation
All pots = -2*Zval/r beyond r=  1.1278

VLOCAL1: 99.0% of the norm of Vloc inside     34.126 Ry
VLOCAL1: 99.9% of the norm of Vloc inside     77.774 Ry
atom: Maximum radius for 4*pi*r*r*local-pseudopot. charge    1.37759
atom: Maximum radius for r*vlocal+2*Zval:    1.18566
GHOST: No ghost state for L =  0
GHOST: No ghost state for L =  1
GHOST: No ghost state for L =  2
GHOST: No ghost state for L =  3

KBgen: Kleinman-Bylander projectors: 
   l= 0   rc=  1.294105   el= -1.742414   Ekb=  9.135903   kbcos=  0.326910
   l= 1   rc=  1.294105   el= -0.676589   Ekb= -8.124878   kbcos= -0.395047
   l= 2   rc=  1.448233   el=  0.002386   Ekb= -2.039267   kbcos= -0.003484
   l= 3   rc=  1.561052   el=  0.003508   Ekb= -0.799141   kbcos= -0.000344

KBgen: Total number of  Kleinman-Bylander projectors:   16
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:
atom: Selected multiple-zeta basis: split     

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 2s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.014700 Ry

   izeta = 1
                 lambda =    1.000000
                     rc =    3.388770
                 energy =   -1.727268
                kinetic =    1.600577
    potential(screened) =   -3.327845
       potential(ionic) =  -11.280754

   izeta = 2
                 rmatch =    2.510382
              splitnorm =    0.150000
                 energy =   -1.454222
                kinetic =    2.481391
    potential(screened) =   -3.935613
       potential(ionic) =  -12.511044

SPLIT: Orbitals with angular momentum L= 1

SPLIT: Basis orbitals for state 2p

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.014700 Ry

   izeta = 1
                 lambda =    1.000000
                     rc =    4.036919
                 energy =   -0.661551
                kinetic =    4.973792
    potential(screened) =   -5.635343
       potential(ionic) =  -13.404116

   izeta = 2
                 rmatch =    2.573941
              splitnorm =    0.150000
                 energy =   -0.383510
                kinetic =    7.441916
    potential(screened) =   -7.825426
       potential(ionic) =  -16.516775

POLgen: Perturbative polarization orbital with L=  2

POLgen: Polarization orbital for state 2p

   izeta = 1
                     rc =    4.036919
                 energy =    2.343864
                kinetic =    4.617795
    potential(screened) =   -2.273931
       potential(ionic) =   -8.509049
atom: Total number of Sankey-type orbitals: 13

atm_pop: Valence configuration (for local Pseudopot. screening):
 2s( 2.00)                                                            
 2p( 4.00)                                                            
Vna: chval, zval:    6.00000   6.00000

Vna:  Cut-off radius for the neutral-atom potential:   4.036919

atom: _________________________________________________________________________

prinput: Basis input ----------------------------------------------------------
GREETS from /home/kovalp/programs/siesta/4.0-b2/Src/atom.F     7006

PAO.BasisType split     

%block ChemicalSpeciesLabel
    1    1 H                       # Species index, atomic number, species label
    2    8 O                       # Species index, atomic number, species label
%endblock ChemicalSpeciesLabel

%block PAO.Basis                 # Define Basis set
H                     1                    # Species label, number of l-shells
 n=1   0   2 P   1                   # n, l, Nzeta, Polarization, NzetaPol
   5.076      3.953   
   1.000      1.000   
O                     2                    # Species label, number of l-shells
 n=2   0   2                         # n, l, Nzeta 
   3.389      2.510   
   1.000      1.000   
 n=2   1   2 P   1                   # n, l, Nzeta, Polarization, NzetaPol
   4.037      2.574   
   1.000      1.000   
%endblock PAO.Basis

prinput: ----------------------------------------------------------------------

coor:   Atomic-coordinates input format  =     Fractional

siesta: Atomic coordinates (Bohr) and species
siesta:      4.26594  -2.46294   2.42006  1        1
siesta:      8.53189  -4.92589   9.37425  1        2
siesta:      4.26594   2.46294   9.37425  1        3
siesta:      0.00000   4.92589   2.42006  1        4
siesta:      0.00000   9.85178   9.37425  1        5
siesta:      8.53189   4.92589   2.42006  1        6
siesta:      5.60545  -3.23631   0.36162  1        7
siesta:      7.19238  -4.15252   7.31581  1        8
siesta:      5.60545   3.23631   7.31581  1        9
siesta:      0.00000   6.47262   0.36162  1       10
siesta:      0.00000   8.30505   7.31581  1       11
siesta:      7.19238   4.15252   0.36162  1       12
siesta:      9.87993  -4.15252  13.56068  1       13
siesta:      2.91791  11.54136   6.60649  1       14
siesta:      8.53615   8.29766  13.56068  1       15
siesta:      8.53615   6.48001   6.60649  1       16
siesta:      1.34377  10.63253  13.56068  1       17
siesta:      4.26168  -0.90883   6.60649  1       18
siesta:      1.34377   4.14513   6.60649  1       19
siesta:      4.26168   0.90883  13.56068  1       20
siesta:     11.45406   3.24370  13.56068  1       21
siesta:     11.45406  -3.24370   6.60649  1       22
siesta:      2.91791   3.23631  13.56068  1       23
siesta:      9.87993   4.15252   6.60649  1       24
siesta:      4.26594  -2.46294   0.86927  2       25
siesta:      8.53189  -4.92589   7.82347  2       26
siesta:      4.26594   2.46294   7.82347  2       27
siesta:      0.00000   4.92589   0.86927  2       28
siesta:      0.00000   9.85178   7.82347  2       29
siesta:      8.53189   4.92589   0.86927  2       30
siesta:      8.53189  -4.92589  13.03912  2       31
siesta:      4.26594  -2.46294   6.08492  2       32
siesta:      8.53189   4.92589   6.08492  2       33
siesta:      0.00000   9.85178  13.03912  2       34
siesta:      0.00000   4.92589   6.08492  2       35
siesta:      4.26594   2.46294  13.03912  2       36

siesta: System type = bulk      

initatomlists: Number of atoms, orbitals, and projectors:     36   276   408

siesta: ******************** Simulation parameters ****************************
siesta:
siesta: The following are some of the parameters of the simulation.
siesta: A complete list of the parameters used, including default values,
siesta: can be found in file out.fdf
siesta:
redata: Non-Collinear-spin run           =     F
redata: SpinPolarized (Up/Down) run      =     T
redata: Number of spin components        =     2
redata: Long output                      =     F
redata: Number of Atomic Species         =        2
redata: Charge density info will appear in .RHO file
redata: Write Mulliken Pop.              =     NO
redata: Mesh Cutoff                      =   100.0000  Ry
redata: Net charge of the system         =     0.0000 |e|
redata: Min. number of SCF Iter          =        0
redata: Max. number of SCF Iter          =     1000
redata: Mix DM or H after convergence    =     F
redata: Recompute H after scf cycle      =     F
redata: Performing Pulay mixing using    =     3 iterations
redata: Mix DM in first SCF step ?       =     F
redata: Write Pulay info on disk?        =     F
redata: Discard 1st Pulay DM after  kick =     F
redata: New DM Mixing Weight             =     0.5000
redata: New DM Occupancy tolerance       = 0.000000000001
redata: No kicks to SCF
redata: DM Mixing Weight for Kicks       =     0.5000
redata: DM Tolerance for SCF             =     0.000100
redata: Require (free) Energy convergence in SCF =     F
redata: DM (free)Energy tolerance for SCF =     0.000010 eV
redata: Require Harris convergence for SCF =     F
redata: DM Harris energy tolerance for SCF =     0.000010 eV
redata: Antiferro initial spin density   =     F
redata: Using Saved Data (generic)   =     F
redata: Use continuation files for DM    =     T
redata: Neglect nonoverlap interactions  =     F
redata: Method of Calculation            =     Diagonalization
redata: Divide and Conquer               =     T
redata: Electronic Temperature           =     0.0019  Ry
redata: Fix the spin of the system       =     F
redata: Dynamics option                  =     Single-point calculation
redata: ***********************************************************************
Total number of electrons:    96.000000
Total ionic charge:    96.000000

* ProcessorY, Blocksize:    2  24


* Orbital distribution balance (max,min):    72    60

k-point displ. along   1 input, could be:     0.00    0.50
k-point displ. along   2 input, could be:     0.00    0.50
k-point displ. along   3 input, could be:     0.00    0.50
 Kpoints in:            8 . Kpoints trimmed:            8

siesta: k-grid: Number of k-points =     8
siesta: k-grid: Cutoff (effective) =     7.360 Ang
siesta: k-grid: Supercell and displacements
siesta: k-grid:    2   0   0      0.000
siesta: k-grid:    0   2   0      0.000
siesta: k-grid:    0   0   2      0.000
Naive supercell factors:     2    2    2

superc: Internal auxiliary supercell:     2 x     2 x     2  =       8
superc: Number of atoms, orbitals, and projectors:    288   2208   3264

                     ====================================
                        Single-point calculation
                     ====================================

superc: Internal auxiliary supercell:     2 x     2 x     2  =       8
superc: Number of atoms, orbitals, and projectors:    288   2208   3264

outcell: Unit cell vectors (Ang):
        6.772319   -3.910000    0.000000
        0.000000    7.820000    0.000000
        0.000000    0.000000    7.360000

outcell: Cell vector modules (Ang)   :    7.820000    7.820000    7.360000
outcell: Cell angles (23,13,12) (deg):     90.0000     90.0000    120.0000
outcell: Cell volume (Ang**3)        :    389.7822
New_DM. Step:     1
Initializing Density Matrix...

initdm: Initial spin polarization (Qup-Qdown) =   48.000000
New grid distribution:   1
           1       1:   24    1:   12    1:   12
           2       1:   24    1:   12   13:   24
           3       1:   24   13:   24    1:   12
           4       1:   24   13:   24   13:   24

InitMesh: MESH =    48 x    48 x    48 =      110592
InitMesh: (bp) =    24 x    24 x    24 =       13824
InitMesh: Mesh cutoff (required, used) =   100.000   117.552 Ry
ExtMesh (bp) on 0 =    64 x    52 x    48 =      159744
New grid distribution:   2
           1       1:   24    1:   12    1:   12
           2       1:   24    1:   12   13:   24
           3       1:   24   13:   24    1:   12
           4       1:   24   13:   24   13:   24
New grid distribution:   3
           1       1:   24    1:   12    1:   12
           2       1:   24    1:   12   13:   24
           3       1:   24   13:   24    1:   12
           4       1:   24   13:   24   13:   24
Setting up quadratic distribution...
ExtMesh (bp) on 0 =    64 x    52 x    48 =      159744
PhiOnMesh: Number of (b)points on node 0 =                 3456
PhiOnMesh: nlist on node 0 =               137592

stepf: Fermi-Dirac step function

siesta: Program's energy decomposition (eV):
siesta: Ebs     =     -1667.935149
siesta: Eions   =      9790.253741
siesta: Ena     =      2137.231927
siesta: Ekin    =      4066.690784
siesta: Enl     =      -639.983366
siesta: DEna    =         0.000019
siesta: DUscf   =         0.000000
siesta: DUext   =         0.000000
siesta: Exc     =     -1387.355385
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =     -5555.359278
siesta: Etot    =     -5613.669761
siesta: FreeEng =     -5613.669761

   scf: iscf   Eharris(eV)      E_KS(eV)   FreeEng(eV)    dDmax  Ef(eV)
   scf:    1    -5555.3593    -5613.6698    -5613.6698  0.96424 -4.1910
timer: Routine,Calls,Time,% =    IterSCF     1       8.404  75.33
   scf:    2    -5612.1648    -5582.3020    -5582.3020  0.12588 -2.9332
   scf:    3    -5595.3220    -5585.6232    -5585.6232  0.07432 -6.0265
   scf:    4    -5594.5807    -5589.6085    -5589.6085  0.01369 -5.3749
   scf:    5    -5594.5927    -5593.0758    -5593.0758  0.00798 -5.3187
   scf:    6    -5594.5945    -5594.6988    -5594.6988  0.00343 -5.3113
   scf:    7    -5594.5947    -5594.6205    -5594.6205  0.00034 -5.3190
   scf:    8    -5594.5947    -5594.6259    -5594.6259  0.00015 -5.3178
   scf:    9    -5594.5947    -5594.6092    -5594.6092  0.00008 -5.3177

SCF Convergence by dMax criterion
max |DM_out - DM_in|:     0.00008113
SCF cycle converged after    9 iterations

Using DM_out to compute the final energy and forces

siesta: E_KS(eV) =            -5594.5947

siesta: E_KS - E_eggbox =     -5594.5947

siesta: Atomic forces (eV/Ang):
----------------------------------------
   Tot    0.000000    0.000000   -0.172234
----------------------------------------
   Max   13.884460
   Res    6.219801    sqrt( Sum f_i^2 / 3N )
----------------------------------------
   Max   13.884460    constrained

Stress-tensor-Voigt (kbar):     -225.88     -225.88     -313.08        0.00        0.00       -0.00
(Free)E + p*V (eV/cell)    -5532.5723
Target enthalpy (eV/cell)    -5594.5947
Writing WFSX for COOP/COHP in siesta.fullBZ.WFSX

siesta: Program's energy decomposition (eV):
siesta: Ebs     =     -1469.244081
siesta: Eions   =      9790.253741
siesta: Ena     =      2137.231927
siesta: Ekin    =      4375.942359
siesta: Enl     =      -827.002659
siesta: DEna    =      -107.933610
siesta: DUscf   =         7.627839
siesta: DUext   =         0.000000
siesta: Exc     =     -1390.206838
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =     -5594.594722
siesta: Etot    =     -5594.594722
siesta: FreeEng =     -5594.594722

siesta: Final energy (eV):
siesta:  Band Struct. =   -1469.244081
siesta:       Kinetic =    4375.942359
siesta:       Hartree =    3028.618004
siesta:    Ext. field =       0.000000
siesta:   Exch.-corr. =   -1390.206838
siesta:  Ion-electron =  -10080.622055
siesta:       Ion-ion =   -1528.326193
siesta:       Ekinion =       0.000000
siesta:         Total =   -5594.594722

siesta: Atomic forces (eV/Ang):
siesta:      1   -0.240724    0.138981   13.884460
siesta:      2    0.240724   -0.138981   13.884460
siesta:      3   -0.240722   -0.138981   13.884457
siesta:      4   -0.000001   -0.277963   13.884460
siesta:      5    0.000001    0.277963   13.884460
siesta:      6    0.240722    0.138981   13.884457
siesta:      7    7.477194   -4.316952   -3.216198
siesta:      8   -7.477194    4.316952   -3.216198
siesta:      9    7.477187    4.316958   -3.216199
siesta:     10    0.000009    8.633916   -3.216197
siesta:     11   -0.000009   -8.633916   -3.216197
siesta:     12   -7.477187   -4.316958   -3.216199
siesta:     13    7.001359    4.064866    2.670379
siesta:     14   -7.001359   -4.064866    2.670379
siesta:     15   -0.019591   -8.095789    2.670376
siesta:     16   -0.019589    8.095793    2.670376
siesta:     17    7.020955    4.030924    2.670379
siesta:     18    0.019591    8.095789    2.670376
siesta:     19    7.020954   -4.030930    2.670377
siesta:     20    0.019589   -8.095793    2.670376
siesta:     21   -7.020954    4.030930    2.670377
siesta:     22   -7.020955   -4.030924    2.670379
siesta:     23   -7.001367    4.064863    2.670377
siesta:     24    7.001367   -4.064863    2.670377
siesta:     25   -7.090916    4.093935  -10.716164
siesta:     26    7.090916   -4.093935  -10.716164
siesta:     27   -7.090917   -4.093944  -10.716161
siesta:     28   -0.000007   -8.187879  -10.716165
siesta:     29    0.000007    8.187879  -10.716165
siesta:     30    7.090917    4.093944  -10.716161
siesta:     31   -6.877130    3.970507   -5.321559
siesta:     32    6.877130   -3.970507   -5.321559
siesta:     33   -6.877134   -3.970513   -5.321555
siesta:     34   -0.000003   -7.941025   -5.321559
siesta:     35    0.000003    7.941025   -5.321559
siesta:     36    6.877134    3.970513   -5.321555
siesta: ----------------------------------------
siesta:    Tot    0.000000    0.000000   -0.172234

siesta: Stress tensor (static) (eV/Ang**3):
siesta:    -0.140979    0.000000    0.000000
siesta:     0.000000   -0.140979    0.000000
siesta:    -0.000000    0.000000   -0.195405

siesta: Cell volume =        389.782155 Ang**3

siesta: Pressure (static):
siesta:                Solid            Molecule  Units
siesta:           0.00173303          0.00159598  Ry/Bohr**3
siesta:           0.15912073          0.14653703  eV/Ang**3
siesta:         254.94226880        234.78074271  kBar
(Free)E+ p_basis*V_orbitals  =       -5586.691465
(Free)Eharris+ p_basis*V_orbitals  =       -5586.691465

siesta: Total spin polarization (Qup-Qdown) =   -0.000000
>> End of run:  10-JUL-2017  19:38:13
